synapse/federation/federation_base.py

```diff
@@ -116,6 +122,9 @@ async def _check_sigs_and_hash(
                         "event_id": pdu.event_id,
                     }
                 )
+                await self._store.record_event_failed_pull_attempt(
+                    pdu.room_id, pdu.event_id, "Event content has been tampered with"
+                )
             return redacted_event
 
         spam_check = await self.spam_checker.check_event_for_spam(pdu)
```

Comment:
We will potentially record an attempt when the spam checker fails an event in a future PR. Part of https://github.com/matrix-org/synapse/issues/13700